File: Week 2/python Programming/prog10.py
def count_zeros(matrix):
    zero_count = 0
    for row in matrix:
        zero_count += row.count(0)
    return zero_count

# Example:
matrix = [
    [1, 0, 3],
    [0, 0, 6],
    [7, 8, 0]
]

print("Number of zeros in the matrix:", count_zeros(matrix))

File: Week 2/C Programming/prog10.c
#include <stdio.h>

int main() {
    int rows, cols;
    printf("Enter rows and columns: ");
    scanf("%d %d", &rows, &cols);

    int mat[rows][cols];
    printf("Enter matrix elements:\n");
    for (int i = 0; i < rows; i++) {
        for (int j = 0; j < cols; j++) {
            scanf("%d", &mat[i][j]);
        }
    }

    int zeroCount = 0;
    for (int i = 0; i < rows; i++) {
        for (int j = 0; j < cols; j++) {
            if (mat[i][j] == 0) {
                zeroCount++;
            }
        }
    }

    printf("Number of zeros in the matrix = %d\n", zeroCount);

    return 0;
}
